examples/tensorflow/distribution_strategy/keras-API/Dockerfile

```
@@ -1,5 +1,7 @@
 FROM python:3.9
 
+RUN apt-get update && apt-get install -y libhdf5-dev
```

```suggestion
RUN apt-get update \
 && apt-get install -y libhdf5-dev \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
```

Should we make the container image lightweight?

---

Absolutely @tenzen-y !! Thanks for the suggestion. I will definitely make these changes and make it light weight.